react-app/src/utils/test-helpers/renderForm.tsx

```diff
@@ -1,27 +1,93 @@
 import { QueryClient, QueryClientProvider } from "@tanstack/react-query";
-import { render } from "@testing-library/react";
-import { ReactElement } from "react";
-import { MemoryRouter, Route, Routes } from "react-router-dom";
+import { render, waitForElementToBeRemoved } from "@testing-library/react";
+import React, { ReactElement } from "react";
+import { MemoryRouter, createMemoryRouter, RouterProvider, RouteObject } from "react-router-dom";
+import items from "mocks/data/items";
 
-export const renderForm = (form: ReactElement) =>
-  render(form, {
+export type RouterOptions = {
+  routes: RouteObject[];
+  options: {
+    initialEntries?: string[];
+    initialIndex?: number;
+  };
+};
```

Comment:
Would it be better to use `Parameters<typeof createMemoryRouter>`? Saves us having to update the type in the future